apps/sms/locales/sms.en-US.properties

```diff
@@ -308,6 +349,19 @@ message-delivery-status-delivered.ariaLabel          = Delivered
 message-delivery-status-read.ariaLabel               = Read
 message-delivery-status-error.ariaLabel              = Error
 
+# Late arrival notices
+late-notice-label.ariaLabel           = Late arrival
+late-arrival-notice-in-months         = {[ plural(lateness) ]}
+late-arrival-notice-in-months[zero]   = {{ lateness }}mth late
+late-arrival-notice-in-months[one]    = {{ lateness }}mth late
+late-arrival-notice-in-months[two]    = {{ lateness }}mths late
+late-arrival-notice-in-months[few]    = {{ lateness }}mths late
+late-arrival-notice-in-months[many]   = {{ lateness }}mths late
+late-arrival-notice-in-months[other]  = {{ lateness }}mths late
+late-arrival-notice-in-days           = {{ lateness }}d late
+late-arrival-notice-in-hours          = {{ lateness }}h late
+late-arrival-notice-in-minutes        = {{ lateness }}m late
```

Comment:
I hope you have appropriate space for that, since this string will probably take much more space in many languages. @flodolo ?

Is there any chance to use `Late arrival: 5 months` instead of `5mths late`? That would also allow you to jump on the `Intl.UnitFormat` for `duration-month-long` instead of custom strings.